generics/constants/common.js

```diff
@@ -66,5 +66,10 @@ module.exports = {
   "LINK" : "link",
   "DISCOVERED_BY_ME" : "discoveredByMe",
   "DEFAULT_SUBMISSION_REQUIRED" : 1,
-  "SCHOOL" : "school"
+  "SCHOOL" : "school",
+  "SUBENTITY" :"subEntity_",
+  "SUNBIRD_SERVER_TIMEOUT" : 5000,
```

Comment:
@VISHNUDAS-tunerlabs add SUNBIRD_SERVER_TIMEOUT, CACHE_TIME_PERIOD in env only if these values are added in env for other services.